### PR TITLE
[AllBundles] Enhancement: Streamline tests

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | ---

### CODE REVIEW

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

**Comment by OskarStark** [comment]
any chance to get this merged? @acrobat

**Comment by OskarStark** [comment]
Oh the tests are failing, I will have a look 👀

File: src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php [hunk 1]
```diff
@@ -6,20 +6,13 @@
 use Kunstmaan\PagePartBundle\Tests\unit\Form\PagePartAdminTypeTestCase;
```
Comment (acrobat) [inline src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php:6]: This use causes the failing tests (maybe we should search all use statements for the old /unit/ namespace)

**Review by acrobat — CHANGES_REQUESTED** [review]

File: src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php (hunk 1 above)
Comment (OskarStark) [reply src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php:6]: Thanks, will do 👍🏻

Comment (OskarStark) [reply src/Kunstmaan/PagePartBundle/Tests/Form/HeaderPagePartAdminTypeTest.php]: Done 👍  Lets see if the tests pass!

**Comment by OskarStark** [comment]
any idea @acrobat ?
```
There were 13 errors:
1) Kunstmaan\TranslatorBundle\Tests\Service\Importer\TranslationRepositoryTest::testGetAllDomainsByLocale
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Repository/TranslationRepositoryTest.php:15
2) Kunstmaan\TranslatorBundle\Tests\Service\Importer\TranslationRepositoryTest::testGetLastChangedTranslationDate
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Repository/TranslationRepositoryTest.php:15
3) Kunstmaan\TranslatorBundle\Tests\Service\Importer\TranslationRepositoryTest::testGetTranslationsByLocalesAndDomains
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Repository/TranslationRepositoryTest.php:15
4) Kunstmaan\TranslatorBundle\Tests\Service\Importer\TranslationRepositoryTest::testFindAllNotDisabled
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Repository/TranslationRepositoryTest.php:15
5) Kunstmaan\TranslatorBundle\Tests\Service\Importer\TranslationRepositoryTest::testFindAllDeprecated
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Repository/TranslationRepositoryTest.php:15
6) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testExecuteImportCommand
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
7) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testdetermineLocalesToImport
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
8) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testParseRequestedLocalesMulti
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
9) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testParseRequestedLocalesSingle
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
10) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testParseRequestedLocalesArray
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
11) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testImportBundleTranslationFiles
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
12) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImportCommandHandlerTest::testImportSf4TranslationFiles
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImportCommandHandlerTest.php:15
13) Kunstmaan\TranslatorBundle\Tests\Service\Importer\ImporterTest::testImportExistingDomainFileForced
InvalidArgumentException: The test case "TranslatorBundleTest" does not exist.
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:36
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/WebTestCase.php:58
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/vendor/symfony/framework-bundle/Test/KernelTestCase.php:79
/home/travis/build/Kunstmaan/KunstmaanBundlesCMS/src/Kunstmaan/TranslatorBundle/Tests/Service/Importer/ImporterTest.php:18
```

**Comment by acrobat** [comment]
👍 I will take a look at the failing tests!

**Comment by OskarStark** [comment]
Thank you!

**Comment by OskarStark** [comment]
Rebased

File: src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php [hunk 2]
```diff
@@ -32,14 +32,14 @@ class AppKernel extends Kernel
 
     public function __construct($varDir, $testCase, $rootConfig, $environment, $debug)
     {
-        if (!is_dir(__DIR__ . '/' . $testCase)) {
+        if (!is_dir(__DIR__.'AppKernel.php/'. $testCase)) {
```
Comment (acrobat) [inline src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:35]: Why did you these (also the one below) changes?

```suggestion
        if (!is_dir(__DIR__.'/'.$testCase)) {
```

File: src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php [hunk 3]
```diff
@@ -32,14 +32,14 @@ class AppKernel extends Kernel
...
             throw new \InvalidArgumentException(sprintf('The test case "%s" does not exist.', $testCase));
         }
         $this->varDir = $varDir;
         $this->testCase = $testCase;
 
         $fs = new Filesystem();
-        if (!$fs->isAbsolutePath($rootConfig) && !file_exists($rootConfig = __DIR__ . '/' . $testCase . '/' . $rootConfig)) {
+        if (!$fs->isAbsolutePath($rootConfig) && !file_exists($rootConfig = __DIR__.'/'. $rootConfig)) {
```
Comment (acrobat) [inline src/Kunstmaan/TranslatorBundle/Tests/app/AppKernel.php:42]:
```suggestion
        if (!$fs->isAbsolutePath($rootConfig) && !file_exists($rootConfig = __DIR__.'/'.$testCase.'/'.$rootConfig)) {
```

**Review by acrobat — CHANGES_REQUESTED** [review]
The suggested changes should fix the tests

**Comment by OskarStark** [comment]
Thanks will check and also rebase on Monday.

**Comment by acrobat** [comment]
@OskarStark the test failure should be fixed in the 5.6 branch, can you rebase your PR one last time? Thanks!

**Comment by OskarStark** [comment]
Sure thing

**Review by acrobat — APPROVED** [review]

**Comment by acrobat** [comment]
Thanks @OskarStark!

**Pull request merged**